hub/apps/get-started/uno-simple-photo-viewer.md

```diff
@@ -0,0 +1,937 @@
+
+Soon enough, you will be able to reap the benefits of this approach, as you can target more platforms with a familiar XAML flavor and the codebase you already have.
+
+Open Visual Studio and create a new project via `File` > `New` > `Project`:
+
+:::image type="content" source="../images/uno/uno-create-project.png" alt-text="Create a new project":::
+
+Search for Uno and select the Uno Platform App project template:
+
+:::image type="content" source="../images/uno/uno-new-project.png" alt-text="Uno platform app":::
+
+Create a new C# solution using the **Uno Platform App** type from Visual Studio's **Start Page**. To avoid conflicting with the code from the previous tutorial, we'll give this solution a different name, "UnoSimplePhotos". Specify the project name, solution name, and directory. In this example, our `UnoSimplePhotos` multi-platform project belongs to a `UnoSimplePhotos` solution, which will live in C:\Projects:
+
+:::image type="content" source="images/uno/uno-configure-project.png" alt-text="Specify project details":::
+
+Now you'll choose a base template to take your Simple Photo gallery application multi-platform.
+
+The Uno Platform App template comes with two preset options that allow you to quickly get started with either a **Blank** solution or the **Default** configuration which includes references to the Uno.Material and Uno.Toolkit libraries. The Default configuration also includes Uno.Extensions which is used for dependency injection, configuration, navigation, and logging, and it uses MVUX in place of MVVM, making it a great starting point for rapidly building real-world applications.
+
+:::image type="content" source="../images/uno/uno-vsix-new-project-options.png" alt-text="Uno solution template for project startup type":::
+
+To keep things simple, select the **Blank** preset. Then, click the **Create** button. Wait for the projects to be created and their dependencies to be restored.
+
+A banner at the top of the editor may ask to reload projects, click **Reload projects**:
+:::image type="content" source="../images/vs2022-project-reload.png" alt-text="Visual Studio banner offering to reload your projects to complete changes":::
+
+You should see the following default file structure in your **Solution Explorer**:
+
+:::image type="content" source="images/uno/screenshot2.png" alt-text="Default file structure":::
+
+### Add image assets to the project
+
+Your app will need some images to display. You can use the same images from the previous tutorial.
+
+In the `UnoSimplePhotos` project, create a new folder named `Assets` and copy the JPG image files to a `Samples` subfolder. The `Assets` folder structure should now look like this:
+
+:::image type="content" source="images/uno/screenshot3.png" alt-text="dotnet mobile workload in VS":::
```

Comment:
Is this the correct alt text for this screenshot? It's duplicated with the one on line 25.